index.bs

```diff
@@ -16,11 +18,21 @@ Abstract: loading process and customizing loading behavior.
 Logo: https://resources.whatwg.org/logo-javascript.svg
 !Version History: <a href="https://github.com/whatwg/loader/commits">https://github.com/whatwg/loader/commits</a>
 !Participate: <a href="https://github.com/whatwg/loader/issues/new">File an issue</a> (<a href="https://github.com/whatwg/loader/issues?state=open">open issues</a>)
+
+Opaque Elements: emu-alg
 </pre>
 
 <style>
-  ol > li { margin: 0; }
+  .note + .example, .note + .note { margin-top: 1em; }
+
+  emu-val { font-weight: bold; }
+  emu-alg > ol, emu-alg > ol ol ol ol { list-style-type: decimal; }
+  emu-alg > ol ol, emu-alg > ol ol ol ol ol { list-style-type: lower-alpha; }
+  emu-alg > ol ol ol, emu-alg > ol ol ol ol ol ol { list-style-type: lower-roman; }
+  emu-alg li { margin: 0; }
 </style>
+<script src="https://resources.whatwg.org/file-issue.js" async></script>
+<script src="https://resources.whatwg.org/commit-snapshot-shortcut-key.js" async></script>
```

Comment:
this is not really needed as right now, but eventually we will probably want to have the snapshots, I can remove it from now I guess.